js/validation.js

```diff
@@ -133,3 +134,31 @@ function redirectToWishlist() {
 
 
 ////////////////////////////  localstorage   //////////////////////////////////////
+
+function saveDatatoLocalStorage() {
+	// get the data that you want to save
+	var name = document.letterToSanta.myName.value;
+	var description = document.letterToSanta.description.value;
+	var key = 'user'+ localStorageIndex // setting a unique key for the data
+
+	// saving the data as an object
+	localStorage.setItem(key, JSON.stringify({
+    username: name,
+    giftDescription: description
+	}));
+
+	localStorageIndex++; // incrementing the index counter
+}
+
+function displayWishes() {
+	var ul = document.getElementById('wish-list');
```

Comment:
Would be better to use `getElementByQuerySelector` and change `id` to `class` for `wish-list`. Better to show good practices from beginning :)